src/google/protobuf/compiler/objectivec/import_writer.cc

```diff
@@ -140,15 +140,11 @@ void ImportWriter::AddFile(const FileDescriptor* file,
     return;
   }
 
-  // Lazy parse any mappings.
-  if (need_to_parse_mapping_file_) {
-    ParseFrameworkMappings();
-  }
+  auto proto_lookup = ModuleForFile(file);
```

Comment:
Lets rename this to "module_name" to better match the result.

Reply:
done